legacy/application/views/scripts/form/preferences_general.phtml

```diff
@@ -46,6 +46,9 @@
         <?php echo $this->element->getElement('thirdPartyApi')->render() ?>
 
         <?php echo $this->element->getElement('radioPageLoginButton')->renderViewHelper() ?>
-        <?php echo $this->element->getElement('radioPageLoginButton')->renderLabel() ?>
+        <?php echo $this->element->getElement('radioPageLoginButton')->renderLabel() . '<p/>' ?>
```

Comment:
Why is the `. '<p/>'` needed ? Please remove it.

Reply:
without the page break the checkboxes are in the same line in the UI. I didn't want to introduce new styling, replacing the tunein style for such a simple change.

Reply:
IF we get all the changes in, i will restyle/update the settings UI to eliminate these classes.